plugins/apache-airflow/README.md

```diff
@@ -0,0 +1,134 @@
+# Apache Airflow Plugin
+
+Welcome to the apache-airflow plugin!
+
+## Installation
+
+1. Clone the plugin repository to the plugins directory.
+
+```sh
+git clone \
+    git@github.com:cmpadden/backstage-apache-airflow-plugin.git \
+    plugins/apache-airflow
+```
+
+2. Add the plugin dependency in `app/package.json`
+
+```diff
+--- a/packages/app/package.json
++++ b/packages/app/package.json
+@@ -11,6 +11,7 @@
+     "@backstage/core-components": "^0.7.4",
+     "@backstage/core-plugin-api": "^1.2.0",
+     "@backstage/integration-react": "^0.1.14",
++    "@backstage/plugin-apache-airflow": "^0.0.0",
+     "@backstage/plugin-api-docs": "^0.6.14",
+     "@backstage/plugin-azure-devops": "^0.1.4",
+     "@backstage/plugin-badges": "^0.2.14",
+```
+
+3. Import and use the plugin extension in `spp/src/App.tsx`
+
+```diff
+--- a/packages/app/src/App.tsx
++++ b/packages/app/src/App.tsx
+@@ -86,6 +86,7 @@ import { providers } from './identityProviders';
+ import * as plugins from './plugins';
+
+ import { techDocsPage } from './components/techdocs/TechDocsPage';
++import { ApacheAirflowPage } from '@backstage/plugin-apache-airflow';
+
+ const app = createApp({
+   apis,
+@@ -203,6 +204,7 @@ const routes = (
+       element={<CostInsightsLabelDataflowInstructionsPage />}
+     />
+     <Route path="/settings" element={<UserSettingsPage />} />
++    <Route path="/apache-airflow" element={<ApacheAirflowPage />} />
+   </FlatRoutes>
+ );
+```
+
+## Configuration
+
+For links to the Airflow instance, the `baseUrl` must be defined in
+`app-config.yaml`.
+
+```yaml
+apacheAirflow:
+  baseUrl: https://your.airflow.instance.com
+```
+
+This plugin uses the Backstage proxy to securely communicate with the Apache
+Airflow API. Add the following to your `app-config.yaml` to enable this
+configuration:
+
+```yaml
+proxy:
+  '/airflow':
+    target: http://localhost:8080/api/v1
```

Comment:
Is this suppose to be `http://localhost:8080/api/v1` or https://your.airflow.instance.com? I'm reviewing top down so please ignore if this is OK 😁

Reply:
Good catch. Updated the default values in `app-config.yml` and the README.